test-apps/appui-test-app/appui-test-providers/src/ui/providers/WidgetApiStageUiItemsProvider.tsx

```diff
@@ -135,7 +143,7 @@ export class WidgetApiStageUiItemsProvider implements UiItemsProvider {
       widgets.push({
         id: "WR-A",
         label: "WR-A",
-        icon: "icon-text-align-text-align-left",
+        icon: <SvgTextAlignLeft />,
```

Comment:
Does it works correctly for webicons ? Or only for SVG icons ?

Reply:
It worked correctly with webicons.